code/datums/antagonists/antag_datum.dm

```diff
@@ -0,0 +1,102 @@
+var/global/list/antagonists = list()
+
+/datum/antagonist
+	var/name = "Antagonist"
+	var/datum/mind/owner						//Mind that owns this datum
+	var/silent = FALSE							//Silent will prevent the gain/lose texts to show
+	var/can_coexist_with_others = TRUE			//Whether or not the person will be able to have more than one datum
+	var/list/typecache_datum_blacklist = list()	//List of datums this type can't coexist with
+	var/delete_on_mind_deletion = TRUE
+	var/antag_role //the antag type
+	var/replace_banned = TRUE //Should replace jobbaned player with ghosts if granted.
+	var/list/objectives = list()
+	var/antag_memory = ""//These will be removed with antag datum
+
```

Comment:
also needs:

```
	//Antag panel properties
	var/show_in_antagpanel = TRUE	//This will hide adding this antag type in antag panel, use only for internal subtypes that shouldn't be added directly but still show if possessed by mind
	var/antagpanel_category = "Uncategorized"	//Antagpanel will display these together, REQUIRED
	var/show_name_in_check_antagonists = FALSE //Will append antagonist name in admin listings - use for categories that share more than one antag type
```

Reply:
Isn't that all related to the antags panel refactor the did that I am pretty sure we do not have?

Reply:
Yeah, but might as well keep it in, for now.